### PR TITLE
Add click component

### PR DESCRIPTION
**Pull request by rch-liu**
Added file cli.py that links with the backend to allow api requests from terminal. It currently supports the following requests:
- register
- login
- create-order
- show-orders
- health-check

To hit an endpoint, execute in your terminal:
- Ensure you have a port open at http://0.0.0.0:8000/
- Type: python cli.py [request] [--arguments]

### CODE REVIEW

File: backend/cli.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    'user_email': user_email,
+    }
+    resp = req.post(url, data=json.dumps(payload))
+
+    click.echo("***Output of POST request on /orders*** \n")
+    click.echo('Response: \n' + (resp.text) + '\n')
+    click.echo('Status Code: \n' + str(resp))
+
+
+messages.add_command(health_check)
```
Comment (roeeyn) [inline backend/cli.py]: I think the CLI should only have `login`,`register`,  `healthcheck`, and `orders` commands, as the other endpoints/options are mainly for food delivery side options.

**Review by roeeyn — CHANGES_REQUESTED** [review]
Could you erase the `.pyc` files from the commit? You can do this by adding `*.pyc` to the `.gitignore` file 💯 , also I think it's good idea to ignore the `.db` file too as we don't want it to appear in the vc history 😄 .
Also I think you may only want to add in this commit the `backend/cli.py` file, as the other files will be updated by the other branch `backend/database-connection`. Maybe it would make sense to create a completely separate folder for the cli part, as it is independent from the `backend` folder.
Great work! 🚀

File: backend/cli.py (hunk 1 above)
Comment (rch-liu) [reply backend/cli.py]: Good point 👍do we want to give users the ability update(PUT) /orders?

Comment (diarreola) [reply backend/cli.py]: I dont think we need to give the users the ability to update(PUT) /orders.

Comment (rch-liu) [reply backend/cli.py]: Yeah that's what i'm thinking as well - i'm going to leave that functionality out

**Review by diarreola — COMMENTED** [review]
Looks good!

**Review by roeeyn — APPROVED** [review]
Nice work! 🚀

**Pull request merged**